components/atoms/tooltip/index.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Popover from 'react-popover'
+
+import styles from './styles.css'
+
+class Tooltip extends React.Component {
+  state = {
+    isOpen: true,
+  }
+
+  toggle(toState = null) {
+    this.setState({ isOpen: toState === null ? !this.state.isOpen : toState })
```

Comment:
I think you can use `prevState` like this code below

``sideMenuHandler = () => {
    this.setState(prevState => ({
      sidenavIsOpen: !prevState.sidenavIsOpen,
    }))
  }
``